R/spark_compile.R

```diff
@@ -126,7 +126,7 @@ spark_compile <- function(name,
 #'
 #' @keywords internal
 #' @export
-spark_compile_package_jars <- function(package = rprojroot::find_package_root_file(),
+compile_package_jars <- function(package = rprojroot::find_package_root_file(),
```

Comment:
I could go either way on the function name as well (technically, we'll be generating one `jar` for each combination of Spark version * scala version provided)